src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -464,6 +466,8 @@ private void GroupStyleCheckmark_Click(object sender, RoutedEventArgs e)
         private void GroupStyleAnnotation_SubmenuOpened(object sender, RoutedEventArgs e)
         {
             ViewModel.ReloadGroupStyles();
+            // Tracking loading group style items
+            Logging.Analytics.TrackEvent(Actions.Load, Categories.GroupStyleOperations, nameof(GroupStyleItem) + "s");
```

Comment:
Why are we adding the hard coded "s" here?

Reply:
@zeusongit This is tracking user hovered over group styles to load all the options, and with 4 defaults ones, user will always see many, so I suggest tracking as `GroupStyleItems`. What do you think?

Reply:
got it, thanks